src/DynamoPackages/PackageLoader.cs

```diff
@@ -71,7 +71,7 @@ public IEnumerable<IExtension> RequestedExtensions
         private readonly List<string> packagesDirectories = new List<string>();
         public string DefaultPackagesDirectory
```

Comment:
hmm - I'm thinking this could start gettng a bit confusing, could we add a /// summary to this property to explain what we mean by default?

Reply:
Will do.